src/components/ReportingTile.vue

```diff
@@ -0,0 +1,164 @@
+        >
+          <img
+            v-if="item.isReported"
+            src="/checkmark.svg"
+            :alt="`${item.year} data reported`"
+            class="reported"
+          >
+          <img
+            v-else
+            src="/cross.svg"
+            :alt="`${item.year} data not reported`"
+          >
+        </div>
+        <p>{{ item.year }}</p>
+      </li>
+    </ul>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Prop, Vue } from "vue-property-decorator";
+import { IHistoricData } from "../common-functions.vue";
+
+import { LatestDataYear } from "../constants/globals.vue";
+
+@Component
+export default class ReportingTile extends Vue {
+  @Prop() historicData?: Array<IHistoricData>;
+
+  readonly LatestDataYear: number = LatestDataYear;
+
+  get grade(): string {
+    const gradeRanges = [
+      { min: 0.9, grade: "A" },
+      { min: 0.8, grade: "B" },
+      { min: 0.7, grade: "C" },
+      { min: 0.6, grade: "D" },
+      { min: 0, grade: "F" },
+    ];
+
+    const score = this.reportedYearsCount / this.reportingHistory.length;
+    return gradeRanges.find((range) => score >= range.min)!.grade;
+  }
+
+  get reportedYearsCount(): number {
+    return this.reportingHistory.filter((entry) => entry.isReported).length;
+  }
+
+  get reportingHistory(): { year: number; isReported: boolean }[] {
+    if (!this.historicData || this.historicData.length === 0) return [];
+
+    const reportingHistory = [];
+    const reportedYears = this.historicData.map((entry) =>
+      parseInt(entry.DataYear),
+    );
+
+    let currentYear = reportedYears[0];
+    for (let reportedYear of reportedYears) {
+      // take care of missing years that lie in between reported years
+      while (currentYear !== reportedYear) {
+        reportingHistory.push({ year: currentYear, isReported: false });
+        currentYear++;
+      }
+      // if year is reported (standard case)
+      reportingHistory.push({ year: currentYear, isReported: true });
+      currentYear++;
+    }
+
+    /**
+     * if a building stopped reporting before the latest year that we have data for,
+     * add the missing years to history and mark them as not reported
+     */
+    while (currentYear <= LatestDataYear) {
+      reportingHistory.push({ year: currentYear, isReported: false });
+      currentYear++;
+    }
+
+    return reportingHistory;
+  }
+}
+</script>
+
+<style lang="scss">
+.reporting-tile {
+  .headline {
+    font-size: 1.5rem;
+    font-weight: bold;
+    margin-right: 0.5rem;
+    margin-bottom: 1rem;
+
+    .score {
+      font-size: 1.1rem;
+      font-weight: bold;
+      margin-right: 1rem;
+    }
+  }
+
+  ul {
+    display: flex;
+    flex-wrap: wrap;
+    gap: 1rem 2rem;
+    list-style: none;
+    padding: 0;
+
+    .reporting-tile-item {
+      margin-top: 0;
+      text-align: center;
+      font-weight: bold;
+    }
+  }
+
+  .marker {
+    position: relative;
+    width: 40px;
```

Comment:
Let's convert this to `rem`, should be `2.5rem` since we're using the default 16px font size